src/App.js

```diff
@@ -18,45 +19,51 @@ const COMPANIES_SOURCE = 'companies';
 const MAPS = CONFIG['maps'];
 const POINT_LAYER = 'energy-companies-point-layer';
 
+// mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_API_TOKEN;
```

Comment:
Remove commented out code

Reply:
Great points! Thank you for mentioning some comments to remove! Both of them are removed  now